modules/nf-core/cnvkit/antitarget/main.nf

```diff
@@ -2,10 +2,10 @@ process CNVKIT_ANTITARGET {
     tag "$meta.id"
     label 'process_low'
 
-    conda "bioconda::cnvkit=0.9.9 bioconda::samtools=1.16.1"
+    conda "bioconda::cnvkit=0.9.10 bioconda::samtools=1.17"
```

Comment:
can you update the depencies table in the CHANGELOG related to this tool?

Reply:
How do you want that written in the dependencies table?  Like this?

```
| Dependency | Old version               | New version              |
| ---------- | ------------------------- | ------------------------ |
| `cnvkit`   | 0.9.9 (`samtools` 1.16.1) | 0.9.10 (`samtools` 1.17) |
| `multiqc`  | 1.14                      | 1.15                     |
| `tiddit`   | 3.3.2                     | 3.6.1                    |
```